README.md

```diff
@@ -1,36 +1,251 @@
+
+## Installation
+
++++++++++++++++++++++++++++++++++++++++
+FOR WINDOWS (tested for Wondows 10 1903)
++++++++++++++++++++++++++++++++++++++++
+Package installation:
+=========================================
+zip Installation:
+=========================================
+1. Run Console with administrator privileges (run as Administrator).
+2. In console go to directory with rss-reader-3.0.zip package.
+3. Run the following command: `pip install rss-reader-3.0.zip`
+=========================================
+zip Package Run:
+=========================================
+1. Run Console with administrator privileges (run as Administrator).
+2. Run the following command: `rss-reader [url] [list of flags]`
+
+
++++++++++++++++++++++++++++++++++++++++
+FOR LINUX (tested for Fedora 30)
++++++++++++++++++++++++++++++++++++++++
+Package installation:
+=========================================
+zip Installation:
+=========================================
+1. Copy `rss-reader-3.0.zip` to desired directory (for example `~/rss-reader`).
+2. In console go to directory with `rss-reader-1.0.zip` package.
+3. Run the following command: `sudo pip3 install rss-reader-3.0.zip`.
+=========================================
+zip Package Run:
+=========================================
+1. In console run the following command: `sudo rss-reader [url] [list of flags]`
+
+
++++++++++++++++++++++++++++++++++++++++
+CREATION OF PACKAGE
+++++++++++++++++++++++++++++++++++++++
+How to create package using setup.py
+======================================
+1. Download RssReader to desired directory
+2. Go to the `.../rss-reader` folder wich has setup.py file
+3.1 Windows: Run console as administrator
+   3.1.1 Run the following code: `python setup.py sdist --formats=zip`
+3.1 Linux: Run the following command: `sudo python3 sdist --formats=zip`
+4. The zip package will be inside `.../rss-reader/dist` folder.
+
+
+## progect structure
+
+RssReader
+|---rss-reader
+|    |
+|    |---args_parser.py
+|    |---json_converter.py
+|    |---logs.py
+|    |---other.py
+|    |---news.py
+|    |---rss_reader.py
+|    |---rss_parser.py
+|    |---requirements.txt
+|    |---tests
+|        |-----test_args_parser.py
+|        |-----test_rss_parser.py
+|
+|---setup.py
+|---README.txt
+|---__init__.py
+
+
+## Resourses for testing
+
+RssReader was tested with following RSS:
+
+http://feeds.bbci.co.uk/news/rss.xml?edition=int
+https://news.yahoo.com/rss/
+https://3dnews.ru/news/rss/
+https://www.tomshardware.com/feeds/all
+https://people.onliner.by/feed
+http://static.feed.rbc.ru/rbc/logical/footer/news.rss
+https://www.gazeta.ru/export/rss/first.xml
+https://rss.nytimes.com/services/xml/rss/nyt/World.xml
+https://www.reddit.com/r/worldnews/.rss
+https://www.aljazeera.com/xml/rss/all.xml
+http://feeds.washingtonpost.com/rss/world
+https://www.engadget.com/rss.xml
+
+
+   
+ 
```

Comment:
Please move your documentation to `final_task/README.md` and checkout any changes for this file.

Reply:
Fixed